cvu/utils/backend_onnx/convert_to_onnx.py

```diff
@@ -0,0 +1,72 @@
+"""
+ONNX conversion from different frameworks.
+"""
+
+from typing import Tuple
+import os
+import onnx
+import torch
+
+
+def onnx_from_torchscript(
+    torchscript_model: str,
+    shape: Tuple[int, int]=(640, 640),
+    save_path=None,
+    dynamic=False):
+    """Convert torchscript model to ONNX.
+    Args:
+        torchscript_model: path to torchscript model
+        shape: input shape of the model
+        save_path: path to save onnx model
+        simplify: bool to simplify onnx model
+        dynamic: bool for onnx dynamic shape conversion
```

Comment:
minor:
- type hint
- removing `simplify` arg doc
```suggestion
        torchscript_model (str): path to torchscript model
        shape (Optional[Tuple[int, int]]): input shape of the model
        save_path (Optional[str]): path to save onnx model
        dynamic (Optional[bool]): bool for onnx dynamic shape conversion
```